app/lib/cooldown.js

```diff
@@ -16,13 +23,15 @@ module.exports = Base => class extends Base {
         //instantiate playerName variable if it hasn't been instantiated yet.
         if (!this.onCooldownMap[playerName]) this.onCooldownMap[playerName] = {}
 
-        //if command is on cooldown then return true.
-        if (this.onCooldownMap[playerName][command] == true) {
+        //if command is on cooldown then return true. typeof will always be a number or false.
+        if (typeof this.onCooldownMap[playerName][command] == "number") {
             this.whisperPlayerRaw(playerName, [
-                { text: `!${command} `, color: 'white' },
-                { text: `has a cooldown of `, color: 'red' },
-                { text: `${this.coodownTimes[command] / 60000} minutes`, color: 'white' },
-                { text: ` total.\nTry again later.`, color: 'red' },
+                { text: `${command} `, color: 'white' },
```

Comment:
I  can see how this would make the wording work a little better.. but you still are not getting the explicit message that would tell you `!cove` is on cool down.

I also don't like that this solution requires the timers to have an `!` in the property name